operations/node_management/Swap_a_Compute_Blade_with_a_Different_System.md

```diff
@@ -106,6 +106,43 @@ Swap an HPE Cray EX liquid-cooled compute blade between two systems.
    cray hsm inventory redfishEndpoints update --enabled false x9000c3s0b1
    ```
 
+### Source: Clear out the existing Redfish event subscriptions from the BMCs on the blade
+
+1. (`ncn#`) Set the environment variable `SLOT` corresponding to the blades location:
+
+     ```bash
+     ncn# SLOT="x9000c3s0"
+     ```
+
+1. (`ncn#`) Clear the Redfish event subscriptions:
+
+    ```bash
+    ncn# for BMC in $(cray hsm inventory  redfishEndpoints list --type NodeBMC --format json | jq .RedfishEndpoints[].ID -r | grep $SLOT); do
+        PASSWD=$(cray scsd bmc creds list --targets $BMC --format json | jq .Targets[].Password -r)
+        SUBS=$(curl -sk -u root:$PASSWD https://${BMC}/redfish/v1/EventService/Subscriptions | jq -r '.Members[]."@odata.id"')
```

Comment:
Would be better to avoid putting the password on the curl command line, because that is visible in the output of `ps -ef` for any users logged into the system, regardless of access level.